src/quacc/calculators/espresso/espresso.py

```diff
@@ -180,7 +234,15 @@ def __init__(
             binary=str(bin_path), parallel_info=parallel_info, pseudo_path=pseudo_path
         )
 
-        super().__init__(profile=profile, parallel_info=parallel_info, **kwargs)
+        if kwargs.pop("directory", None):
+            warn(
+                "It is highly discouraged to use the 'directory' parameter when using quacc."
+            )
```

Comment:
Let's change this to a raised error since it would do absolutely gnarly things. Can simply say it's not supported.